### PR TITLE
Rhode Island Standard Deduction

### LINKED ISSUE #2343: Rhode Island Standard Deduction
reference: https://tax.ri.gov/sites/g/files/xkgbur541/files/2022-12/ADV_2022_40_Inflation_Adjustments.pdf
page 1.

### PR DESCRIPTION
**Pull request by SchizoidMann**
Fixes #2343

Co-authored-by: Shuyu Zhan (Stella) <shuyu7zhan@users.noreply.github.com>

### Summary
:sparkles::page_facing_up::white_check_mark:

This pull request adds a new feature to the policy engine: the Rhode Island standard deduction. It creates a new parameter, variable, and test file for this feature, and updates the changelog accordingly.

> _`RI_std_deduction`_
> _New feature for tax-calc_
> _Autumn leaves fall fast_

### Walkthrough
*  Add Rhode Island standard deduction feature ([link](https://github.com/PolicyEngine/policyengine-us/pull/2344/files?diff=unified&w=0#diff-5885042762833a4aef51405a4036067e4e6aefa7a5091a2987d1d9d1fc2647e8R1-R4))
   - Create parameter file for standard deduction amount by filing status and year ([link](https://github.com/PolicyEngine/policyengine-us/pull/2344/files?diff=unified&w=0#diff-1f7963b3c87b2adc8c176e0944cb2539aa1717fb64de13a3c2b208ba42fe2946R1-R23))
   - Create variable file for standard deduction formula based on parameter values and tax unit attributes ([link](https://github.com/PolicyEngine/policyengine-us/pull/2344/files?diff=unified&w=0#diff-7012c63d3d33108cbedf27b6617cefa8af88a956ae2b8e693c841689bae38869R1-R16))
   - Create test file for standard deduction output values for different filing statuses and years ([link](https://github.com/PolicyEngine/policyengine-us/pull/2344/files?diff=unified&w=0#diff-9273df7b377bac99ece8622d60d0cffc839872412e39350558608d5ac072d30cR1-R32))

### CODE REVIEW

File: policyengine_us/parameters/gov/states/ri/tax/income/deductions/standard/amount.yaml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+description: Rhode Island provides filers a standard deduction of this amount, depending on filing status.
+metadata:
+  label: Rhode Island standard deduction
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/ri/tax/income/deductions/standard/amount.yaml:3]:
```suggestion
  label: Rhode Island standard deduction amount
```

File: policyengine_us/parameters/gov/states/ri/tax/income/deductions/standard/amount.yaml
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/ri/tax/income/deductions/standard/amount.yaml:1]: Could you please add the 'WIDOW" case? (same amounts as joint)

File: policyengine_us/parameters/gov/states/ri/tax/income/deductions/standard/amount.yaml [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  period: year
+  unit: currency-USD
+  breakdown:
+  - filing_status
+  reference:
+  - title: Inflation-adjuested amounts set for tax year 2023.
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/ri/tax/income/deductions/standard/amount.yaml:9]:
```suggestion
  - title: Rhode Island Department of Revenue, Division of Taxation, Inflation-adjusted amounts set for tax year 2023.
```

File: policyengine_us/tests/policy/baseline/gov/states/ri/tax/income/deductions/standard/ri_standard_deduction.yaml [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+- name: Head of household filer
+  period: 2023
+  input:
+    state_code: RI
+    filing_status: HEAD_OF_HOUSEHOLD
+  output:
+    ri_standard_deduction: 15_050
+
+- name: Married filing separately filer
```
Comment (PavelMakarchuk) [inline policyengine_us/tests/policy/baseline/gov/states/ri/tax/income/deductions/standard/ri_standard_deduction.yaml:25]:
```suggestion
- name: Married filing separately 
```

File: policyengine_us/variables/gov/states/ri/tax/income/deductions/standard/ri_standard_deduction.py [hunk 4]
```diff
@@ -0,0 +1,16 @@
+from policyengine_us.model_api import *
+
+
+class ri_standard_deduction(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Rhode Island standard deduction"
+    unit = USD
+    documentation = " https://tax.ri.gov/sites/g/files/xkgbur541/files/2022-12/ADV_2022_40_Inflation_Adjustments.pdf"
```
Comment (MaxGhenis) [inline policyengine_us/variables/gov/states/ri/tax/income/deductions/standard/ri_standard_deduction.py:9]:
```suggestion
    documentation = "https://tax.ri.gov/sites/g/files/xkgbur541/files/2022-12/ADV_2022_40_Inflation_Adjustments.pdf"
```

**Review by MaxGhenis — APPROVED** [review]

**Pull request merged**